### ISSUE TITLE
Accessibility: Settings categories list items have no listbox container.

### ISSUE THREAD

**Issue by MarcoZehe** [microsoft/terminal#9568]
### Windows Terminal version (or Windows build number)

Windows Terminal Preview Version: 1.7.572.0

### Other Software

Screen readers like Narrator, NVDA, JAWS

### Steps to reproduce

1. Turn on your screen reader of choice.
2. In Terminal, open Settings. You'll land on the Close Navigation button.
3. Tab once to focus the list of Settings categories.

### Expected Behavior

Screen reader would say "Listbox", item name, "1 of 6", or similar.

### Actual Behavior

It speaks the item names, but not that these are contained within a list box. This information is necessary, though, to know what navigation mode to use (arrow keys, for example).

Examining the accessibility tree, it shows that there is no listbox container, unlike in the Windows Settings app, for example. The category items are direct children of the Settings window instead.

**Comment by zadjii-msft** [comment]
I wonder, does this repro with the `NavigationView` in the Xaml Controls Gallery app? This seems like something that should be provided by default by that control.

**Comment by MarcoZehe** [comment]
Got a link to that app? I'm happy to check it out. I definitely know that the Standard Windows Settings app has this right.

**Comment by zadjii-msft** [comment]
https://www.microsoft.com/en-us/p/xaml-controls-gallery/9msvh128x2zt

**Comment by MarcoZehe** [comment]
Found it, thanks, and it has the same problem for vertically aligned NavigationView items: https://github.com/microsoft/Xaml-Controls-Gallery/issues/658.

**Comment by zadjii-msft** [comment]
Oh maybe I wasn't clear 😬  Since it does repro in the controls gallery, then the place to file it upstream is https://github.com/microsoft/microsoft-ui-xaml/issues - that's the source for all the WinUI controls. Both the Terminal and the Xaml Controls Gallery are using standard `NavigationView`s, so this is probably a bug with the underlying implementation.

**Comment by MarcoZehe** [comment]
Filed: https://github.com/microsoft/microsoft-ui-xaml/issues/4609

**Comment by Don-Vito** [comment]
@DHowett - it seems that microsoft-ui-xaml are not planning to fix this, stating it is not an issue: https://github.com/microsoft/microsoft-ui-xaml/issues/4609#issuecomment-807186502

**Comment by zadjii-msft** [comment]
Alright well, I'll trust that they've done the diligence on this one.

/dup https://github.com/microsoft/microsoft-ui-xaml/issues/4609#issuecomment-807186502
